core/domain/skill_mastery_services_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Filename should match the thing it is testing. If this tests skill_services then the test file name should be skill_services_test.

Reply:
My bad, Sorry